src/platform/macos.rs

```diff
@@ -438,7 +437,7 @@ pub trait EventLoopBuilderExtMacOS {
 impl EventLoopBuilderExtMacOS for EventLoopBuilder {
     #[inline]
     fn with_activation_policy(&mut self, activation_policy: ActivationPolicy) -> &mut Self {
```

Comment:
Please document the new behaviour, so that it's clear that calling `with_activation_policy` will _overwrite_ the value.